src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs

```diff
@@ -532,6 +532,20 @@ private ResourceManifest BuildManifest()
             .SetDependencies("monaco-loader")
             .SetVersion(MonacoEditorVersion);
 
+        manifest
+            .DefineScript("jquery-nestedSortable")
```

Comment:
Also please move this up to the other jQuery plugins.

```suggestion
            .DefineScript("jQuery.nestedSortable")
```

Reply:
done